teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
Please add `parent_permission = true` as the parameter in the constructor of `Teacher` class and pass it as the argument to the constructor of superclass like `super(age, name, parent_permission)`.